decidim-elections/spec/system/admin/admin_manages_voting_polling_stations_spec.rb

```diff
@@ -173,6 +173,8 @@
         )
         fill_in :polling_station_address, with: address
 
+        find("#autoComplete_list_1 > li").click
```

Comment:
Why are adding this change?

Reply:
Sometimes the spec failing due to fact that some elements cannot be clicked. This is caused by the fact the Test is filling very fast the fields, and in this particular case, the autocomplete list is being drawn over the pop-up itself. 

![image](https://github.com/decidim/decidim/assets/105683/cdc91d21-47b1-47ed-bdfe-4cba0532e975)

Reply:
Great, makes perfectly sense!